README.md

```diff
@@ -179,4 +179,4 @@ Questions can be asked by raising an `Issue`.
 - [Vinicius Ramos](https://github.com/redrookie)
 - [Vivek Singh](https://github.com/vivmost)
 - [Victory Chiamaka Wekwa](https://github.com/VictoryWekwa)
-
+- [Meenal Jain](https://github.com/meenal21)
```

Comment:
Looks good 👍 , could you please add it alphabetically to the list, then I can merge

Reply:
Hey, now I have put my name in the alphabetically and committed, could you please check?

Reply:
Looks good, merged